src/components/Sidenav/Sidenav.tsx

```diff
@@ -97,7 +97,7 @@ const Sidenav: React.FC<SidenavProps> = ({ items, isStudio }) => {
               variant="secondary"
               onClick={closeSideNav}
               icon="external"
-              to={isStudio ? routes.index() : routes.studio()}
+              to={isStudio ? routes.viewer.index() : routes.studio.index()}
```

Comment:
Can't go back to viewer view, this will need something like `../` or maybe separate route for going back from studio to viewer instead of `routes.viewer.index()`

Reply:
@mikkio-j are you sure? Works for me

Reply:
My bad, sorry. You were right, can't go back

Reply:
Fixed now. @drillprop your suggestion would indeed solve it but it would create another problem here:
```ts
if (
  cachedLocation.state?.overlaidLocation?.pathname === location.pathname ||
  location.pathname === [routes.viewer.index(), routes.viewer.search()].join('/')
)
```
Sometimes we will need to operate on those routes so ended up with a different approach